src/amo/components/Categories/styles.scss

```diff
@@ -14,15 +14,34 @@
 }
 
 .Categories-item {
-  @include margin-end(5px);
+  @include margin-end(7px);
 
   display: inline-block;
-  margin: 0 0 5px;
+  margin: 0 0 7px;
```

Comment:
Does these two values (`7px`) are somehow related? If so, maybe we could have a variable.